src/other/Table.js

```diff
@@ -469,18 +503,22 @@
             }
             this.fixedCol
                 .style("position", "absolute")
-                .style("margin-top", -this.tableDiv.node().scrollTop + tableMarginHeight + "px")
+                .style("margin-top", -this.tableDiv.node().scrollTop + parseInt(tableMarginHeight) + "px")
             ;
             fixedColTd
                 .style("width", fixedColWidth + "px")
             ;
             this.fixedColHead
                 .style("position", "absolute")
-                .style("margin-top", (this.fixedHeader() ? this.tableDiv.node().scrollTop: 0) - tableMarginHeight + "px")
+                .style("margin-top", (this.fixedHeader() ? this.tableDiv.node().scrollTop: 0) - parseInt(tableMarginHeight) + "px")
             ;
             fixedColTh
                 .style("width", fixedColWidth + "px")
             ;
+            rows.each(function(d, i) {
+                var height = d3.select(this).select("td").node().offsetHeight;
```

Comment:
Technically it can't since in this case I'm using "select" instead of "selectAll", so it only returns the first `td`.  You can see an embedded table in the demo http://rawgit.com/dtsnell4/Visualization/GH-1161_Table_Child_Widgets/demos/dermatology.html?src/other/Table.widget